drivers/input/touchscreen/rpi-ft5406.c

```diff
@@ -61,13 +60,14 @@ static int ft5406_thread(void *arg)
 	struct ft5406 *ts = (struct ft5406 *) arg;
 	struct ft5406_regs regs;
 	int known_ids = 0;
-	
+        uint8_t temp = 99;	
+
 	while(!kthread_should_stop())
 	{
 		// 60fps polling
 		msleep_interruptible(17);
-		memcpy_fromio(&regs, ts->regs, sizeof(*ts->regs));
-		writel(99, &ts->regs->num_points);
+		memcpy_fromio(&regs, ts->ts_base, sizeof(regs));
```

Comment:
I was referring to the first of the two lines (changing ``ts->regs`` to ``ts->base`` and ``sizeof(*ts->regs)`` to ``sizeof(regs)``). Was that just cosmetic or is it actually changing behaviour?